client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -29,17 +31,27 @@ const useStyles = makeStyles({
 interface ProfilePhotoProps {
   header: string;
   currentUser?: User;
-  currentProfile?: any;
+  currentProfile?: Profile;
 }
 
 const ProfilePhoto: React.FC<ProfilePhotoProps> = ({ header, currentUser, currentProfile }) => {
   const fileInputButton = React.useRef<HTMLInputElement>(null);
   const classes = useStyles();
   const { updateSnackBarMessage } = useSnackBar();
+  const { updateProfileContext } = useAuth();
 
   const fileChangeHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
     if (event.target.files) {
-      updateSnackBarMessage('Profile Photo uploaded successfuly!!!');
+      profilePhotoUpload(event.target.files[0]).then((data) => {
```

Comment:
The catch block in the end is recommended to catch errors and is a recommended practice.

Ditto below